cri/cri_test.go

```diff
@@ -56,20 +56,20 @@ func TestSingleInvoke(t *testing.T) {
 	invoke(t, functionURL)
 }
 
-func TestSingleInvokeLocal(t *testing.T) {
-	functionURL := getFuncURL("helloworldlocal")
-	invoke(t, functionURL)
-}
+// func TestSingleInvokeLocal(t *testing.T) {
```

Comment:
add an entry to the changelog saying that we excluded local registry tests for now, raise an issue to keep track and add the issue number next to the commented code